qcodes/instrument_drivers/Keysight/keysightb1500/KeysightB1517A.py

```diff
@@ -622,6 +650,8 @@ def _set_current(self, value: float) -> None:
         )
         self.write(msg.message)
 
+        self.current._measurement_status = None
```

Comment:
@jenshnielsen @lakhotiaharshit shall i extract a parameter class with these get/set methods in order to make it clearer and NOT access private _measurement_status attribute in the B1517A CMU class?

Reply:
@jenshnielsen @lakhotiaharshit just finished the exctraction - could you have another look?